src/test/java/com/aws/greengrass/componentmanager/KernelConfigResolverTest.java

```diff
@@ -481,66 +481,6 @@ void GIVEN_deployment_with_artifact_and_interpolate_config_set_WHEN_config_resol
                 equalTo("java -jar " + "{artifacts:{randomNameSpace:" + artifactsPath + "}}" + "/test.jar -x arg"));
     }
 
-    @Test
```

Comment:
Keep this test, just disable it with a junit annotation

Reply:
Changed it to Disabled.